### PR TITLE
Use babylon instead of babel

### PR DESCRIPTION
**Pull request by forivall**

### CODE REVIEW

**Comment by hzoo** [comment]
Nice catch - it's a good idea since I don't think we need anything else except that at least for now.

**Comment by hzoo** [comment]
Do you think we would want to use everything in `babel-core` parse method? https://github.com/babel/babel/blob/master/packages/babel/src/api/node.js#L81-L106

**Comment by forivall** [comment]
I initially used the arguments from an older version of babel-core, but it was nearly the same. I think it's only es6/es2015+features that need to be manually added

**Comment by hzoo** [comment]
Yeah I asked sebmck to add https://github.com/babel/babel/blob/master/packages/babel/src/api/node.js#L97-L103 for babel-eslint so I guess that's not necessary. So I guess it's only `opts.ecmaVersion = Infinity;` and `opts.allowHashBang = true;` I'l wait for @mdevils to merge - it's :+1: for me.

**Comment by hzoo** [comment]
I guess the script may need to be changed in babel 6 but that's fine

**Comment by mdevils** [comment]
Thank you very much! Great job!

File: src/Parser.js [hunk 1]
```diff
@@ -26,14 +26,35 @@ export default class Parser {
     }
 
     _parseAst(code) {
-        let tokens = [];
+        // let tokens = [];
```
Comment (hzoo) [inline src/Parser.js]: remove this and https://github.com/cst/cst/pull/64/files#diff-a0c375bdbb580536aa70722d5d376137R56 later

**Comment by hzoo** [comment]
awesome

File: src/Parser.js (hunk 1 above)
Comment (mdevils) [inline src/Parser.js]: OK, thanks

**Comment by forivall** [comment]
At the time, i forgot about this repo: https://github.com/hax/babel-features . It _could_ be used to get the options instead of inlining them.

**Pull request merged**